website/static/js/registrationUtils.js

```diff
@@ -260,10 +260,15 @@ var MetaSchema = function(params) {
     self.fulfills = params.fulfills || [];
     self.messages = params.messages || {};
 
+    var count = 0;
     $.each(self.schema.pages, function(i, page) {
         var mapped = {};
         $.each(page.questions, function(qid, question) {
-            mapped[qid] = new Question(question, qid);
+            // This is necessary now since page.questions is a list
```

Comment:
I am happy to write up some documentation on what the schemas are expected to look like. Where should it go?

Re identifers: this is introduced in this PR. Before questions in a schema were an object with a unique key but this caused problems with questions being sorted as well as throwing off the left-side nav ba behavior. Do you have a specific suggestion on an alternative way to approach this?